solana-spl-owner-foundational-store/README.md

```diff
@@ -0,0 +1,14 @@
+# Solana SPL Owner Foundational Store
+
+This foundational module provides the `AccountOwner` message for Solana SPL token account ownership data.
+
+## Message Definition
+
+```protobuf
+message AccountOwner {
+  bytes mint = 1;
+  bytes owner = 2;
+}
+```
+
+This module can be imported and used by other substreams that need to work with account ownership data in the Solana ecosystem.
```

Comment:
Please add `substreams.yaml` manifest  quick example of how to import it. Also document the fact that on their next `substreams build`, Protobuf <package.id>.AccountOwner is available and showcase Rust code on how to quickly use it.